fdb-relational-core/src/main/java/com/apple/foundationdb/relational/recordlayer/RecordLayerIterator.java

```diff
@@ -83,6 +83,7 @@ public boolean hasNext() {
 
     private void fetchNextResult() {
         if (result != null) {
+            continuation = ContinuationImpl.fromRecordCursorContinuation(result.getContinuation());
```

Comment:
What's this about? This change is a little bit notable given that it would affect more than just aggregate queries

Reply:
removed.